form/form.js

```diff
@@ -0,0 +1,28 @@
+import { checkAuth, createPosts, logout } from '../fetch-utils.js';
+
+checkAuth();
```

Comment:
Our checkAuth function returns the unauthenticated user to the home page, however, we want to reroute to the authentication page. Otherwise great work Josh. Awesome job styling! Remember to add those tests in the future too, as testing will be required in future modules :)